ui/src/app/pages/data/concept/concept-table-v2.tsx

```diff
@@ -223,14 +223,14 @@ interface Props {
   searchTerm?: string;
   selectedConcepts: any[];
   error: boolean;
-  concept: Array<Concept>;
+  concept: Array<any>;
 }
 
 interface State {
   first: number;
   pageLoading: boolean;
   growlVisible: boolean;
-  selectedConcepts: Array<Concept>;
+  selectedConcepts: Array<any>;
```

Comment:
Should this type change to `Criteria` instead of any?

Also, does this table go away once we switch to the CB search?

Reply:
Yes is not used anywhere will delete this

Reply:
Done